pages/datamodel.vue

```diff
@@ -0,0 +1,435 @@
+<template>
+  <v-layout column>
+    <h1 class="display-4 ma-5">DataModel</h1>
+
+    <p :class="paragraph">
+      In this guide, you will learn fundamental elements to understand and manipulate OpenGeode data model.
+      One of OpenGeode central classes is
+      <span
+        v-html="brep"
+      />, let's use this class as an exemple.
+    </p>
+
+    <img src="@/assets/datamodel-1.svg" />
+
+    <h2 :class="section">Manipulating Surfaces</h2>
+
+    <p :class="paragraph">
+      A
+      <span v-html="brep" />, for Boundary Representation, represents a 3D object by its boundaries.
+      In OpenGeode, these 3D boundaries are
+      <span
+        v-html="surfaces3d"
+      /> (in grey on images).
+      A BRep is thus composed of Surfaces:
+      it means that the class
+      <span
+        v-html="brep"
+      /> inherits from the class
+      <span v-html="surfaces3d" />.
+      This allows you to know the number of Surfaces in a
+      <span v-html="brep" /> :
+    </p>
+
+    <!-- eslint-disable -->
+    <code class="cpp primary--text">
+      BRep my_brep;
```

Comment:
I know that the purpose is to illustrate the method nb_surfaces, but in this BRep instance there are no surfaces... Maybe use BRep::nb_surfaces().

Reply:
I agree with you, but it was just for illustrating on objects.